### PR TITLE
CDAP-9180 Fix log

### PR DESCRIPTION
**Pull request by deepakwadhwani**
Move to level Error.

### CODE REVIEW

**Comment by CuriousVini** [comment]
Is the fix related to: https://issues.cask.co/browse/CDAP-9180?

File: cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/workflow/WorkflowProgramController.java [hunk 1]
```diff
@@ -95,7 +95,7 @@ public void terminated(Service.State from) {
 
       @Override
       public void failed(Service.State from, Throwable failure) {
-        LOG.info("Workflow service failed from {}. Un-registering service {}.", from, serviceName, failure);
+        LOG.error("Workflow service '{}' failed.", serviceName, failure);
```
Comment (CuriousVini) [inline cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/workflow/WorkflowProgramController.java:98]: why did we remove `Un-registering service ` ?

Comment (deepakwadhwani) [reply cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/workflow/WorkflowProgramController.java:98]: "service unregistered" is a info message underneath. Also "unregistering service" is not really part of the error.

**Comment by deepakwadhwani** [comment]
Yes, I have the Jira in the heading of the PR.

**Comment by CuriousVini** [comment]
Got it. lgtm

**Comment by deepakwadhwani** [comment]
Build https://builds.cask.co/browse/CDAP-DRC6573-1

**Pull request merged**